deploy/olm-catalog/manifests/extensions.hive.openshift.io_agentclusterinstalls.yaml

```diff
@@ -308,6 +308,31 @@ spec:
                       true or leave empty.
                     type: boolean
                 type: object
+              platformType:
+                allOf:
+                - enum:
+                  - ""
+                  - AWS
+                  - Azure
+                  - BareMetal
+                  - GCP
+                  - Libvirt
+                  - OpenStack
+                  - None
+                  - VSphere
+                  - oVirt
+                  - IBMCloud
+                  - KubeVirt
+                  - EquinixMetal
+                - enum:
+                  - ""
+                  - BareMetal
+                  - None
+                  - VSphere
+                  - oVirt
+                description: PlatformType is the name for the specific platform upon
+                  which to perform the installation.
+                type: string
               provisionRequirements:
```

Comment:
After discussions with @avishayt: we should add the actual PlatfromType and user managed networking to the status because the BE may change / set the values of these fields according to the combination of requested Platform, UMN and HA mode